vms/platformvm/utxo/camino_locked.go

```diff
@@ -248,16 +248,37 @@ func (h *handler) Lock(
 		locked   uint64
 		remained uint64
 	}
+	type OwnerID struct {
+		owners   *secp256k1fx.OutputOwners
+		ownersID *ids.ID
+	}
 	type OwnerAmounts struct {
 		amounts map[ids.ID]lockedAndRemainedAmounts
 		owners  secp256k1fx.OutputOwners
-		signers []*crypto.PrivateKeySECP256K1R
 	}
 	// Track the amount of transfers and their owners
 	// if appliedLockState == bond, then otherLockTxID is depositTxID and vice versa
 	// ownerID -> otherLockTxID -> AAAA
 	insAmounts := make(map[ids.ID]OwnerAmounts)
 
+	var toOwnerID *ids.ID
+	if to != nil && appliedLockState == locked.StateUnlocked {
+		if id, err := GetOwnerID(OwnedWrapper{*to}); err != nil {
+			return nil, nil, nil, err
+		} else {
+			toOwnerID = &id
+		}
+	}
+
+	var changeOwnerID *ids.ID
+	if change != nil {
+		if id, err := GetOwnerID(OwnedWrapper{*change}); err != nil {
+			return nil, nil, nil, err
+		} else {
```

Comment:
else {..} could be omitted cause of return in if block

Reply:
same^^